src/main/kotlin/io/sip3/salto/ce/rtpr/RtprHandler.kt

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright 2018-2019 SIP3.IO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.sip3.salto.ce.rtpr
+
+import io.micrometer.core.instrument.DistributionSummary
+import io.micrometer.core.instrument.Metrics
+import io.micrometer.core.instrument.Timer
+import io.netty.buffer.Unpooled
+import io.sip3.commons.domain.payload.RtpReportPayload
+import io.sip3.commons.util.format
+import io.sip3.salto.ce.RoutesCE
+import io.sip3.salto.ce.USE_LOCAL_CODEC
+import io.sip3.salto.ce.domain.Packet
+import io.vertx.core.AbstractVerticle
+import io.vertx.core.json.JsonObject
+import mu.KotlinLogging
+import java.time.format.DateTimeFormatter
+import java.util.concurrent.TimeUnit
+
+open class RtprHandler : AbstractVerticle() {
```

Comment:
Let's add a description for this class. Something simple like: `Handles RTP reports`